src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesSynchronisationService.kt

```diff
@@ -159,20 +159,32 @@ class CaseNotesSynchronisationService(
       }
   }
 
+  suspend fun repairDeletedCaseNote(nomisCaseNoteId: Long) = CaseNotesEvent(
+    caseNoteId = nomisCaseNoteId,
+    offenderIdDisplay = null,
+    caseNoteType = null,
+    caseNoteSubType = null,
+    bookingId = null,
+    auditModuleName = null,
+    recordDeleted = null,
+  ).also {
```

Comment:
I normally use `also` for side effects e.g. logging.  Would be better to use `apply` since it indicates an action.  Also wondering if `CaseNotesEvent` should actually default to `null` for the other fields, rather than having to pass in `null`